app/sessionStore.js

```diff
@@ -233,6 +233,8 @@ module.exports.cleanAppData = (data, isShutdown) => {
   data.temporarySiteSettings = {}
   // Delete Flash state since this is checked on startup
   delete data.flashInitialized
+  // Delete Recovery status on shut down
+  delete data.ui.about.preferences.recoverySucceeded
```

Comment:
i think you just need to make sure this one particular line doesn't cause `Cannot convert undefined or null to object` errors, so a try-catch would be fine